File: python/ex2/plotData.py
import matplotlib.pyplot as plt

# 用两种不同的颜色，在同一张图像上，绘制两个类别的数据散点图
def plot_data(X,Y):
	plt.figure();
	pos = Y == 1
	neg = Y == 0
	plt.scatter(X[neg,0],X[neg,1],c='black',marker='o',s=20)
	plt.scatter(X[pos,0],X[pos,1],c='red',marker='o',s=20)

File: matlab/ex2/plotData.m
function plotData( X, Y)
%% 函数功能：绘制数据中的正负类别
            
%find(x=m)：找到x中值为1的下标向量
pos = find(Y ==1);
neg = find(Y ==0);
plot(X(pos,1),X(pos,2),'k+','LineWidth',2,'MarkerSize',7);
hold on 
plot(X(neg,1),X(neg,2),'ko','MarkerFaceColor','yellow','MarkerSize',7);

end
